### DISCUSSION TITLE
Add a browser backend for theia-blueprint to call the application on localhost:3000.

### DISCUSSION THREAD

**Discussion by frabervo** [eclipse-theia/theia#12108] [Q&A] [answered]
Hello, 
I have downloaded theia-blueprint from the github and wonder how I can add a browser backend server to be able to reach the application online via localhost:3000. 
I don't have much knowledge in nodejs unfortunately. 
Is there a site where I can find out or has someone already done it?

**Comment by msujew** [comment, marked as answer]
Hey @frabervo,

The [release version](https://theia-ide.org/docs/blueprint_download) of Theia blueprint is desktop only. You can try the browser version out over at https://try.theia-cloud.io/. It runs the same codebase as the desktop version. If you're looking for instructions on how to build and run blueprint from source check out its [readme](https://github.com/eclipse-theia/theia-blueprint#development).
